tap_google_search_console/streams/sites.py

```diff
@@ -0,0 +1,36 @@
+from .abstract import FullTableStream
+from tap_google_search_console.helpers import encode_and_format_url, transform_json
+from singer.logger import get_logger
+from typing import Iterator, Dict
+
+LOGGER = get_logger()
+
+
+class Sites(FullTableStream):
+    """
+    Class Representing the `Sites` Stream
+    """
+
+    tap_stream_id = "sites"
+    key_properties = ["site_url", ]
+
+    data_key = "site_entry"
+    path = "sites/{}"
+
+    def __init__(self, client=None, config=None) -> None:
```

Comment:
Remove this __init__ block. It's redundant.

Reply:
Removed